src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -70,6 +70,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_DefaultUserProfileRuntimeStorePath Condition="$([MSBuild]::IsOSPlatform(`Windows`))">$(USERPROFILE)</_DefaultUserProfileRuntimeStorePath>
     <_DefaultUserProfileRuntimeStorePath>$([System.IO.Path]::Combine($(_DefaultUserProfileRuntimeStorePath), '.dotnet', 'store'))</_DefaultUserProfileRuntimeStorePath>
     <UserProfileRuntimeStorePath Condition="'$(UserProfileRuntimeStorePath)' == ''">$(_DefaultUserProfileRuntimeStorePath)</UserProfileRuntimeStorePath>
+    <!-- Prevent including the PredefinedCulturesOnly in the runtime config file in .NET 5.0 and below even if it is defined -->
+    <PredefinedCulturesOnly Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp' Or $([MSBuild]::VersionLessThan($(TargetFrameworkVersion), '6.0'))"></PredefinedCulturesOnly>
```

Comment:
@dsplaisted I added this line too. Please let me know if it make sense to you.

Reply:
I don't believe this is necessary and think it should be removed. If someone is setting this property in a previous TFM, it won't do any harm. It can get in the runtime config for a .NET 5 app, and nothing will read it.

Reply:
That make sense too. I was trying to weigh if it will be useful to have. I just run into that when I was adding the test case. I'll remove it then.